cli/cmd/sync_v3.go

```diff
@@ -424,6 +406,27 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 		pipelineByDestinationName[destinationName] = pipeline
 	}
 
+	// Add a ticker to update the progress bar every 100ms
+	t := time.NewTicker(100 * time.Millisecond)
+	newResources := int64(0)
+	go func() {
+		for {
+			select {
+			case <-gctx.Done():
```

Comment:
Was it deliberate to change the context variable here? This adds the errgroup with all the pipeline goroutines, which is cancelled even when there's no error (i.e. when the goroutines finish). This change isn't necessarily a bad thing, but the code was stopping when the original ctx closed, which was mainly when the user SIGTERM'd.

Reply:
Happy to approve if this is fine, let me know. I didn't because you added automerge.

Reply:
It was deliberate as we should stop the progress bar as soon as the sync finishes (regardless if there's an error or not). Otherwise the timer keeps ticking on an interval and can update the progress after the sync Go routine ended